### PR TITLE
added build step and links to installing node and git

### PR DESCRIPTION
**Pull request by urbantumbleweed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,52 @@
 # ChromeIDE
 Parallax microcontroller development environment based on Chrome application technologies.
 
-## Installing from Source
+## Build from Source
 
-1. Clone this respository.  Choose your file destination from Terminal and
+This is a one-time procedure needed to initially run the application.  To build the application will need the Node javascript library.  If you don't have it, then:
```
Comment (monteslu) [inline README.md]: second sentence missing pronoun?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,52 @@
...
+
+1. Go to the project root directory with
+
+  ``` cd  ChromeIDE/ ```
+
+1. Get all the project dependencies
+
+  ``` npm install ```
+
+1. Bundle and build the application by entering
```
Comment (monteslu) [inline README.md]: Build and bundle

File: README.md [hunk 3]
```diff
@@ -1,32 +1,52 @@
...
 
 	``` chrome://extensions ```
 
-3. Enable 'Developer Mode' by clicking the checkbox.
+1. Enable 'Developer Mode' by clicking the checkbox.
 
   ![enable developer mode](readme-images/enable-developer-mode.png)
 
-4. Click on __'Load unpacked extensions...'__, go to the directory where ChromeIDE was cloned, highlight the *'ChromeIDE'* folder and and click select.
+1. Before you install your extension must be bundled and built.  If you haven't done so follow the instructions in the [Build from Source](#user-content-build-from-source) section first.
```
Comment (monteslu) [inline README.md]: built and bundled

Comment (phated) [inline README.md]: also, the link should just be [#build-from-source] I believe

File: README.md (hunk 1 above)
Comment (phated) [inline README.md]: [`node`](https://nodejs.org/) JavaScript runtime

File: README.md [hunk 4]
```diff
@@ -1,32 +1,52 @@
...
 Parallax microcontroller development environment based on Chrome application technologies.
 
-## Installing from Source
+## Build from Source
 
-1. Clone this respository.  Choose your file destination from Terminal and
+This is a one-time procedure needed to initially run the application.  To build the application will need the Node javascript library.  If you don't have it, then:
 
-	``` git clone git@github.com:parallaxinc/ChromeIDE ```
+1. Install Node by going to [nodejs.org/download](https://nodejs.org/download/) and selecting the option that is right for your system.
```
Comment (phated) [inline README.md]: node is lowercase.

Comment (phated) [inline README.md]: We are going to need to mention node-gyp and it's system-specific dependencies.  See https://github.com/TooTallNate/node-gyp#installation - note that it comes bundled with node/npm so node-gyp doesn't need to be installed but the dependencies do.

File: README.md [hunk 5]
```diff
@@ -1,32 +1,52 @@
...
+1. Clone this respository.  Assuming you have `git` installed (if not, [go here](http://git-scm.com/book/en/v2/Getting-Started-Installing-Git)), choose your file destination from the command-line (CLI like Terminal), and
 
+  ``` git clone git@github.com:parallaxinc/ChromeIDE ```
 
-2. Open your Chrome browser and navigate to
+  ![clone repo](readme-images/clone-chromeIDE.png)
+
+1. Go to the project root directory with
+
+  ``` cd  ChromeIDE/ ```
```
Comment (phated) [inline README.md]: single ` on both sides is fine here

File: README.md [hunk 6]
```diff
@@ -1,32 +1,52 @@
...
+
+1. Get all the project dependencies
+
+  ``` npm install ```
```
Comment (phated) [inline README.md]: single ` on both sides is fine here

File: README.md [hunk 7]
```diff
@@ -1,32 +1,52 @@
...
+  ``` git clone git@github.com:parallaxinc/ChromeIDE ```
 
-2. Open your Chrome browser and navigate to
+  ![clone repo](readme-images/clone-chromeIDE.png)
+
+1. Go to the project root directory with
+
+  ``` cd  ChromeIDE/ ```
+
+1. Get all the project dependencies
```
Comment (phated) [inline README.md]: mention that the dependencies are outlined in package.json - you can take a lot of that info from https://github.com/iceddev/getting-started/blob/master/environments/nodejs-and-npm.md

File: README.md [hunk 8]
```diff
@@ -1,32 +1,52 @@
...
+
+  ``` npm install ```
+
+1. Bundle and build the application by entering
+
+  ```npm run build```
```
Comment (phated) [inline README.md]: single ` on both sides is fine here, doesn't need indent

File: README.md [hunk 9]
```diff
@@ -1,32 +1,52 @@
...
+
+
+## Installing in Chrome
+
+1. Open your Chrome browser and navigate to
 
 	``` chrome://extensions ```
```
Comment (phated) [inline README.md]: single ` on both sides is fine here, doesn't need indent

File: README.md [hunk 10]
```diff
@@ -1,32 +1,52 @@
 # ChromeIDE
 Parallax microcontroller development environment based on Chrome application technologies.
 
-## Installing from Source
+## Build from Source
```
Comment (phated) [inline README.md]: Building from Source / Developing

File: README.md (hunk 8 above)
Comment (phated) [inline README.md]: add a note about `npm run build -- --watch` to to build on any change during development

File: README.md [hunk 11]
```diff
@@ -1,32 +1,52 @@
...
-4. Click on __'Load unpacked extensions...'__, go to the directory where ChromeIDE was cloned, highlight the *'ChromeIDE'* folder and and click select.
+1. Before you install your extension must be bundled and built.  If you haven't done so follow the instructions in the [Build from Source](#user-content-build-from-source) section first.
+
+1. Click on __'Load unpacked extensions...'__, go to the directory where ChromeIDE was built, highlight the *'ChromeIDE'* folder and and click select.
 
 	![load extension](readme-images/load-unpacked-extensions.png)
 	![select folder](readme-images/select-extension-folder.png)
 
-5. Return to the ``` chrome://extensions ``` page and you should see __'Parallax Chrome IDE'__ listed in available extensions.
+1. Return to the ``` chrome://extensions ``` page and you should see __'Parallax Chrome IDE'__ listed in available extensions.
```
Comment (phated) [inline README.md]: single ` on both sides is fine here

File: README.md (hunk 4 above)
Comment (urbantumbleweed) [inline README.md]: I see the installation steps at the link you provided.  Why exactly is node-gyp needed?  Are we using it?

File: README.md [hunk 12]
```diff
@@ -1,32 +1,78 @@
...
-3. Enable 'Developer Mode' by clicking the checkbox.
+  ` git clone git@github.com:parallaxinc/ChromeIDE `
+
+  ![clone repo](readme-images/clone-chromeIDE.png)
+
+1. Go to the project root directory with
+
+  ` cd  ChromeIDE/ `
+
+1. Get all the project dependencies.  At the root of the project you will find `package.json'.  This manifest file includes a list of project dependencies needed to build the application. To learn more, [see this post on dependency installation](https://github.com/iceddev/getting-started/blob/master/environments/nodejs-and-npm.md#user-content-dependency-installation).
```
Comment (phated) [inline README.md]: you used ' instead of ` to close package.json

**Comment by phated** [comment]
I cleaned up a little bit.  Merging now.

**Pull request merged**